scripts/refresh-after-snapshot.py

```
@@ -0,0 +1,780 @@
+                 capture=True).stdout.strip()
+    subnet_prefix = ".".join(node_ip.split(".")[:3])
+    print(f"  MetalLB: {subnet_prefix}.240-{subnet_prefix}.250")
+
+    pool_yaml = json.dumps({
+        "apiVersion": "metallb.io/v1beta1",
+        "kind": "IPAddressPool",
+        "metadata": {"name": "caas-address-pool", "namespace": "metallb-system"},
+        "spec": {"addresses": [f"{subnet_prefix}.240-{subnet_prefix}.250"], "autoAssign": True},
+    })
+    def _try_apply_pool() -> bool:
+        r = subprocess.run(
```

Ruff (0.15.17)

[warning] `subprocess.run` without explicit `check` argument

Add explicit `check=False`

(PLW1510)